SIP/sip-41.md

```diff
@@ -22,6 +22,11 @@ Essentially we suggest the following changes:
  - The account making the distribution is excluded from the distribution
  - All other holdings, including those locked in sell offers are accounted for in the distribution
 
+The same applies to distributions by smart contracts. The difference being, a distribution by regular transaction will account for the
+holdings as they were in the previous block.
+The distribution by a smart contract will use the current state (present block) for the token positions, since smart contract transactions
+are applied last in a block.
```

Comment:
proposal :

The distribution by a smart contract will use the current state (present block) ,**after all transaction in this block are processed** for the token positions, since smart contract transactions are applied last in a block.

Reply:
This is not a change, just a clarification on how it already works. So I would avoid "will use".

Reply:
This comment was removed, should be added as a note to the original SIP on the contracts as this is not being changed.